src/transform-applier.js

```diff
@@ -385,6 +385,7 @@ const _createGradient = function (gradientId, svgTag, bbox, matrix) {
     } else {
         origin = getPoint(newGradient, 'x1', 'y1', false, scaleToBounds);
         destination = getPoint(newGradient, 'x2', 'y2', false, scaleToBounds, '1');
+        if (origin.x === destination.x && origin.y === destination.y) return null;
```

Comment:
Oh, good find!

Reply:
Oh no, it's not what the browser does! The browser seems to have used the first stop
![image](https://user-images.githubusercontent.com/2855464/49903648-e2682d00-fe35-11e8-8965-340948536e24.png)

Reply:
![image](https://user-images.githubusercontent.com/2855464/49903711-1e02f700-fe36-11e8-8186-1825e8b30154.png)
Noooooooooooooooooooooo

Reply:
ooooooooooooooookay I'm just going with whichever

Reply:
I've switched to using the color of the last stop, but I'm going to ignore handling the opacity, because I really don't want to pull another property into this mess